apps/src/search/vue-search-app-full-page.js

```diff
@@ -0,0 +1,9 @@
+import Vue from "vue";
+import SearchFullSite from "./SearchFullSite.vue";
+import VueCompositionAPI from "@vue/composition-api";
+Vue.use(VueCompositionAPI);
+
+new Vue({
+  el: "#search-app-full-page",
```

Comment:
```suggestion
  el: "#search-app-full-site",
```
Shall we renamed this to full-site? I think this naming means exactly what it is doing

Reply:
ah okay yea I think should do that